isos/bootstrap/bootstrap

```diff
@@ -70,36 +79,26 @@ if timeout --signal=KILL 2m mount -t ext4 /dev/disk/by-label/containerfs ${MOUNT
 
     cp /bin/tether ${MOUNTPOINT}/.tether/tether
     cp /bin/unpack ${MOUNTPOINT}/.tether/unpack
+    ln -s tether ${MOUNTPOINT}/.tether/tether-debug
 
     echo 'tether tmpfs size before copying libraries: '
     df -k ${MOUNTPOINT}/.tether
 
-    # ldd of xtables-multi yields the following list of libraries we need to
-    # copy into our initrd.  We need these binaries in order to call iptables
-    #
-    # before the switch-root.  linux-vdso.so.1 (0x00007ffc94d0d000)
-    # libip4tc.so.0 => /baz/lib/libip4tc.so.0 (0x00007f97fc721000)
-    # libip6tc.so.0 => /baz/lib/libip6tc.so.0 (0x00007f97fc519000)
-    # libxtables.so.11 => /baz/lib/libxtables.so.11 (0x00007f97fc30c000)
-    # libm.so.6 => /lib64/libm.so.6 (0x00007f97fc00e000)
-    # libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00007f97fbdf7000)
-    # libc.so.6 => /baz/lib/libc.so.6 (0x00007f97fba53000)
-    # libdl.so.2 => /baz/lib/libdl.so.2 (0x00007f97fb84f000)
-    # /lib64/ld-linux-x86-64.so.2 (0x00007f97fc929000)
-
-    cp -a /sbin/*tables* ${MOUNTPOINT}/.tether/
-    cp -a /lib/libm.* /lib/libm-* /lib/libgcc_s* /lib/libip*tc* /lib/libxtables* /lib/libdl* /lib/libc.so* /lib/libc-* ${MOUNTPOINT}/.tether/lib
-    cp -a /lib64/ld-* ${MOUNTPOINT}/.tether/lib64
-    cp -r /usr/lib/iptables ${MOUNTPOINT}/.tether/usr/lib/
-    cp /lib/libhavege.so.1  ${MOUNTPOINT}/.tether/lib
-    cp /usr/sbin/haveged ${MOUNTPOINT}/.tether/
+    install-iptables ${MOUNTPOINT}/.tether
+    install-entropy ${MOUNTPOINT}/.tether
+
+    echo 'tether tmpfs size after copying libraries: '
+    df -k ${MOUNTPOINT}/.tether
 
     # Create VIC chain
     iptables -N VIC
     # Set the default policy on all chains to drop traffic
     iptables -P INPUT DROP
     iptables -P OUTPUT DROP
     iptables -P FORWARD DROP
+    iptables -P INPUT ACCEPT
```

Comment:
Need to remove these ACCEPT rules - I assume they are only for debugging?

Reply:
Correct.